static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.jsx

```diff
@@ -90,45 +91,50 @@ const SpatialSelectionDropdown = (props) => {
         <Dropdown.Item
           className="spatial-selection-dropdown__button"
           as={Button}
-          icon="edsc-icon-poly-open edsc-icon-fw"
+          icon="edsc-icon-poly edsc-icon-fw"
           onClick={() => onItemClick('polygon')}
           label="Select Polygon"
+          aria-label="Select Polygon"
```

Comment:
I don't think we need labels or aria labels on all of these these dropdown items, since they are text buttons and the text tells the user what they will be selecting